README.md

```diff
@@ -36,15 +38,33 @@ docker run -p 80:8080 \
 cnwan_adaptor
 ```
 
+## Quickstart
+If you want a minimal working setup, the script [setup_kubecon_demo.sh](examples/setup_kubecon_demo.sh) sets everything up a for you in the adaptor and your SD-WAN controller. Before running it, please:
 
+* Install the bash utility `jq`
+* Edit the file [policies_definition.json](examples/policies_definition.json) with the tunnels, vpns and sites of your deployment. You can add as many polcies and SLAs as you need. In addition, specify the metadata keys and values used by the CNWAN reader.
 
 ## SDWAN controller configuration
 
-The adaptor makes the following assumptions regarding the controller configuration:
 
-1. The ```policyName``` variable in the ```mappings``` schema corresponds to a previously defined policy in the controller, containing its tunnel or SLA.
-2. The ```sdwanMergedPolicyName``` variable in the ```credentials``` schema  corresponds to another traffic policy that will be used to merge all the policies into a single one, so it can be activated.
-3. The mapping of ```metadataValue``` to ```policyName``` is 1:1 (two metadata values cannot share the same ```policyName```). On the other hand, a single ```metadataKey``` supports any number of ```metadataValue```.
+
+The adaptor supports both vManage [Application Aware Routing](https://www.cisco.com/c/en/us/td/docs/routers/sdwan/configuration/policies/vedge/policies-book/application-aware-routing.html) (SLAs) and [Traffic Data](https://www.cisco.com/c/en/us/td/docs/routers/sdwan/configuration/policies/vedge/policies-book/data-policies.html#id_107580) (send traffic to a specific tunnel color) policies.
```

Comment:
Let's use the links for version 20 of SD-WAN instead of 19: https://www.cisco.com/c/en/us/td/docs/routers/sdwan/configuration/policies/vedge-20-x/policies-book/application-aware-routing.html and https://www.cisco.com/c/en/us/td/docs/routers/sdwan/configuration/policies/vedge-20-x/policies-book/data-policies.html